bundles/org.openhab.voice.googlestt/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,67 @@
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="voice:googlestt">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Authentication for connecting to Google Cloud Platform.</description>
+		</parameter-group>
+		<parameter-group name="stt">
+			<label>STT Configuration</label>
+			<description>Configure Speech to Text.</description>
+		</parameter-group>
+		<parameter-group name="messages">
+			<label>Info Messages</label>
+			<description>Configure service information messages.</description>
+		</parameter-group>
+		<parameter name="clientId" type="text" required="true" groupName="authentication">
+			<label>Client Id</label>
+			<description>Google Cloud Platform OAuth 2.0-Client Id.</description>
+		</parameter>
+		<parameter name="clientSecret" type="text" required="true" groupName="authentication">
+			<context>password</context>
+			<label>Client Secret</label>
+			<description>Google Cloud Platform OAuth 2.0-Client Secret.</description>
+		</parameter>
+		<parameter name="oauthCode" type="text" groupName="authentication">
+			<label>Authorization Code</label>
+			<description><![CDATA[The oauth code is a one-time code needed to retrieve the necessary access token from Google Cloud Platform. <b>Please go to your browser ...</b> https://accounts.google.com/o/oauth2/auth?client_id=\<YOUR_CLIENT_ID\>&redirect_uri=urn:ietf:wg:oauth:2.0:oob&scope=https://www.googleapis.com/auth/cloud-platform&response_type=code <b>... to generate an auth-code and paste it here</b>.]]></description>
+		</parameter>
+		<parameter name="singleUtteranceMode" type="boolean" groupName="stt">
+			<label>Single Utterance Mode</label>
+			<description>When enabled Google Cloud Platform is responsible for detect when to stop listening after a single
+				utterance. (Recommended)</description>
+			<default>true</default>
+		</parameter>
+		<parameter name="maxTranscriptionSeconds" type="integer" unit="s" groupName="stt">
+			<label>Max Transcription Seconds</label>
+			<description>Max seconds to wait to force stop the transcription.</description>
+			<default>60</default>
+		</parameter>
+		<parameter name="maxSilenceSeconds" type="integer" unit="s" groupName="stt">
+			<label>Max Silence Seconds</label>
+			<description>Only works when singleUtteranceMode is disabled, max seconds without get new transcriptions to stop
+				listening.</description>
+			<default>5</default>
+		</parameter>
+		<parameter name="refreshSupportedLocales" type="boolean" groupName="stt">
+			<label>Refresh Supported Locales</label>
+			<description>Try to load supported locales from the documentation page.</description>
+			<default>false</default>
+		</parameter>
+		<parameter name="noResultsMessage" type="text" groupName="messages">
+			<label>No Results Message</label>
+			<description>Message to be told when no results. (Empty for disabled)</description>
+			<default>Sorry, I didn't understand you</default>
```

Comment:
Having an English hardcoded default value is a problem.
I believe you can't use `@text` in a default tag.
I have no clear idea how to provide a translatable default value.

Reply:
Will be better if I remove the default value?

Reply:
Let me think how to do that properly

Reply:
@cweitkamp / we need your advice here. This is the case we need a default parameter value that should be translated for each language. I believe we can't use `@text.xxx` as value because this is not managed by the core framework ?
What would be the proper solution in this case ?

Reply:
Or we can simply keep the default as it is and users wanting other language than English will update this setting.

Reply:
I think we are okay with the English text as default value here as the user is able to define their own value anyways.

Reply:
Yes, ok for me